pkg/manifest/manifest.go

```diff
@@ -344,6 +344,11 @@ func (pd *ParameterDefinition) Validate() error {
 		pdCopy.ContentEncoding = "base64"
 	}
 
+	// Validate the Parameter Definition schema itself
+	if _, err := pdCopy.Schema.ValidateSchema(); err != nil {
+		return multierror.Append(result, err)
```

Comment:
This error should be wrapped to provide more context on what failed. I think it will be confusing without saying that the schema for parameter X is invalid, or something like that.